cmd/lncli/walletrpc_active.go

```diff
@@ -412,6 +412,14 @@ var listSweepsCommand = cli.Command{
 			Name:  "verbose",
 			Usage: "lookup full transaction",
 		},
+		cli.IntFlag{
+			Name: "startheight",
+			Usage: `The start height to use when fetching sweeps.
```

Comment:
Let's write an itest to validate the description here? I'm curious about the `-1` value here as we already use `btcwallet.UnconfirmedHeight` as our end height so it means the unconfirmed txes are always included?

Reply:
done